src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -223,6 +223,11 @@
     @Argument(doc = "If set, process no more than this many tiles (used for debugging).", optional = true)
     public Integer TILE_LIMIT;
 
+    @Argument(doc ="If set, process only the tile number given and append the tile number to the output file name.",
+            mutex = "FIRST_TILE",
```

Comment:
I forget if there's a convention for this, but should you `mutex` on both options? It's not required but might make it easier for a future maintainer.